src/main/java/org/broadinstitute/consent/http/health/SendGridStatus.java

```diff
@@ -0,0 +1,55 @@
+package org.broadinstitute.consent.http.health;
+
+import com.codahale.metrics.health.HealthCheck.Result;
+import com.google.gson.Gson;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class SendGridStatus {
+    private Object page;
+    private Object status;
```

Comment:
Here, we do need more clarity around what a `status` really is so this should be a class with `indicator` and `description` fields. We also discussed `indicator` being an enum so we don't have to do raw string comparisons.